test/test_defVariable.jl

```diff
@@ -40,10 +40,10 @@ struct NotAManifold end
 @test getDimension(TestVarType2()) === 6
 
 @test getPointType(TestVarType1()) == Vector{Float64}
-@test getPointType(TestVarType2()) == ProductRepr{Tuple{Vector{Float64}, Matrix{Float64}}}
+# @test getPointType(TestVarType2()) == ProductRepr{Tuple{Vector{Float64}, Matrix{Float64}}}
 
 @test getPointIdentity(TestVarType1()) == zeros(3)
-@test getPointIdentity(TestVarType2()).parts[1] == ProductRepr(zeros(3), diagm(ones(3))).parts[1]
-@test getPointIdentity(TestVarType2()).parts[2] == ProductRepr(zeros(3), diagm(ones(3))).parts[2]
+@test getPointIdentity(TestVarType2())[1] == ArrayPartition(zeros(3), diagm(ones(3)))[1]
```

Comment:
This should be .x in place of .parts. Or use submanifold_components

Reply:
ah thanks

Reply:
```suggestion
@test getPointIdentity(TestVarType2()).x[1] == ArrayPartition(zeros(3), diagm(ones(3))).x[1]
```